kubeflow/core/jupyterhub.libsonnet

```diff
@@ -1,342 +1,345 @@
 {
-  parts(params):: [
-    {
-      apiVersion: "v1",
-      kind: "ConfigMap",
-      metadata: {
-        name: "jupyterhub-config",
-        namespace: params.namespace,
+  all(_params):: self + {
+    params+: _params,
+  },
+  list:: [
+    self[key]
+    for key in std.objectFieldsAll(self)
+    if key != "list" && key != "params" && key != "all"
+  ],
+
+  ConfigMap:: {
```

Comment:
nit: Change the name and StatefulSet name to be more descriptive 
e.g
ConfigMap -> KubeSpawnerConfig

StatefulSet -> HubStatefulSet